includes/Core/Modules/Module.php

```diff
@@ -816,4 +816,24 @@ function( $item ) {
 	public static function is_force_active() {
 		return false;
 	}
+
+	/**
+	 * Checks whether the module is shareable.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return bool True if module is shareable, false otherwise.
+	 */
+	public function is_shareable() {
+		if ( $this instanceof Module_With_Owner && $this->is_connected() ) {
+			$definitions = $this->get_datapoint_definitions();
+			foreach ( $definitions as $modules => $details ) {
```

Comment:
Keys in the definitions list are a conjunction of HTTP methods and datapoint names like `GET:report`. So calling them `$modules` doesn't make sense. Would better if we use `$datapoint` name because it gives an exact clue of what it is. However we won't need keys in this loop and can just omit it:

```php
foreach ( $definitions as $details ) {
    ...
}
```